docs/docs/install/docker-compose.mdx

```diff
@@ -54,22 +52,28 @@ Optionally, you can enable hardware acceleration for machine learning and transc
   <CodeBlock language="bash">{ExampleEnv}</CodeBlock>
 </details>
 
-- Populate custom database information if necessary.
 - Populate `UPLOAD_LOCATION` with your preferred location for storing backup assets.
 - Consider changing `DB_PASSWORD` to a custom value. Postgres is not publically exposed, so this password is only used for local authentication.
   To avoid issues with Docker parsing this value, it is best to use only the characters `A-Za-z0-9`.
 - Set your timezone by uncommenting the `TZ=` line.
+- Populate custom database information if necessary.
+
+:::info Optional Features
+You can make the required edits to `docker-compose.yml` to add external libraries or enable hardware acceleration now by following [their guides](#setting-up-optional-features) up to and including those edits.
+:::
 
-### Step 3 - Start the containers
+## Step 3 - Start the containers
 
-From the directory you created in Step 1, (which should now contain your customized `docker-compose.yml` and `.env` files) run `docker compose up -d`.
+From the directory you created in Step 1 (which should now contain your customized `docker-compose.yml` and `.env` files), run `docker compose up -d`.
 
 ```bash title="Start the containers using docker compose command"
 docker compose up -d
 ```
 
+The `restart: always` entries in `docker-compose.yml` for each container mean they will always restart after system reboot or program crashes.
```

Comment:
This isn't needed here (or really at all IMO, this is `man docker` stuff)

Reply:
The purpose of this line is to inform those of us who don’t use or care about Docker that the “compose up” command starts immich as a persistent service, which IMO is a worry that would cross the minds of users who aren’t culturally immersed in Linux to the point of intuitively expecting the behaviour. 

The point isn’t to explain “restart: always” but rather to say “this command starts the container and ensures it automatically restarts”. I’ll reword it to make that clearer.

Reply:
A topic we find ourselves constantly trying to balance is how much basic information to include in our docs. We're documenting Immich, and so I don't think we should expect to be teaching people sysadmin basics. If someone is wondering about details of the restart behaviour, they can find answers to that in plenty of places already.

Reply:
I changed it earlier to something closer to what I intended:

> This starts immich and ensures it automatically restarts after system reboots or crashes (thanks to the `restart: always` entries in `docker-compose.yml`).

The bracketed bit is more for curiosity and can be deleted cos it’s somewhat evident from the non-bracketed bit, if one also reads the yaml file.

But yeah I wouldn’t consider this teaching as much as explaining what that command actually does. On further thought, I recall seeing people forget the d flag, which is kind of a similar mistake – we’re trying to say the command starts the program as a service, so not blocking the console, but as a background thing that auto restarts. Basically that’s what I wanted.

Reply:
New sentence: I am feeling:

> This starts immich as a background service, ensuring it restarts after system reboots or crashes.

Or with btw teaching:

> This starts immich as a background service (per the -d flag), ensuring it restarts after system reboots or crashes (per the `restart` fields in `docker-compose.yml`).